third_party/lua-yaml/lyaml.cc

```diff
@@ -623,6 +618,9 @@ static int dump_node(struct lua_yaml_dumper *dumper)
    bool unused;
    (void) unused;
 
+   yaml_char_t *anchor = get_yaml_anchor(dumper);
+   if (anchor && !*anchor) return 1;
```

Comment:
(It looks a bit strange that `return` is on the same line as `if`.)

Reply:
It's because I copied-and-pasted this piece of code. Fixed:

```diff
diff --git a/third_party/lua-yaml/lyaml.cc b/third_party/lua-yaml/lyaml.cc
index bbbfb677063a..762d85bca59b 100644
--- a/third_party/lua-yaml/lyaml.cc
+++ b/third_party/lua-yaml/lyaml.cc
@@ -632,7 +632,8 @@ static int dump_node(struct lua_yaml_dumper *dumper)
    (void) unused;
 
    yaml_char_t *anchor = get_yaml_anchor(dumper);
-   if (anchor && !*anchor) return 1;
+   if (anchor && !*anchor)
+      return 1;
 
    int top = lua_gettop(dumper->L);
    luaL_checkfield(dumper->L, dumper->cfg, top, &field);
```